### DISCUSSION TITLE
Feature: Allow pinning directories to the sidebar in the File browser

### DISCUSSION THREAD

**Discussion by steelbucks** [IceWhaleTech/CasaOS#229] [Ideas]
I've been playing around with CasaOS for a couple weeks now and have been working to get a NAS system up and running. I noticed that the native "File browser" in the CasaOS web interface doesn't allow pinning of folders on the sidebar. This would be super helpful!

**Comment by AuthorShin** [comment]
Would be great to have this also with on drive share for SMB network share as well or even external FTP to list and connect.

**Reply by Lauren-ED209** [reply]
It's absolutely necessary!

**Reply by kodxana** [reply]
It also would be nice to auto see connected drives in sidebar to fast jump in or at least dropdown menu to select drive.

**Reply by Lauren-ED209** [reply]
> It also would be nice to auto see connected drives in sidebar to fast jump in or at least dropdown menu to select drive.

Copy that.
